buildlib/azure-pipelines-release.yml

```diff
@@ -110,4 +110,6 @@ stages:
     condition: eq(dependencies.Prepare.outputs['CheckRelease.Result.Launch'], 'True')
     jobs:
       - template: az-distro-release.yml
+        parameters:
+          target: publish-release
```

Comment:
should it be "package" for PullRequest and "publish-release" only for IndividualCI?

Reply:
It won't be triggered on PR, since on line 110 there's: ` eq(dependencies.Prepare.outputs['CheckRelease.Result.Launch'], 'True')` that should be triggered only for release

Reply:
actually, this is also for PR which contains the title 'RELEASE'

Reply:
add indent?